src/lib/eina/eina_sched_posix.c

```diff
@@ -21,7 +21,7 @@
 #include <sys/time.h>
 #include <sys/resource.h>
 #include <errno.h>
-#include <pthread.h>
+
```

Comment:
It shouldn't include "eina_thread.h" too?

Reply:
it include eina_sched.h, who includes eina_thread.h

Reply:
Hmm ok. For now it is not clear to me if it is a good practice. Personally I prefer to explicitly include all the needed headers even if it seems redundant..

Reply:
To be honest, personally I would prefer to include wanted dependencies explicitly, but no big deal for now.